projects/frontend/data-pipelines/gui/projects/data-pipelines/package.json

```diff
@@ -53,7 +53,7 @@
     "lodash": "~4.17.21",
     "lottie-web": "~5.9.4",
     "lz-string": "~1.4.4",
-    "moment": "~2.29.3",
+    "moment": "~2.29.4",
```

Comment:
Why does moment gets pinned to 2.29.3? Is there any other moment references?

Reply:
Just the ones that are listed here. We should have a broader discussion about how we handle dependancies anyway. It's something we inherited from Taurus and I'm not sure what the best practice is.